src/Dfe.PlanTech.Web/Controllers/ContentController.cs

```diff
@@ -17,8 +17,7 @@ public class ContentController(
     public const string ErrorActionName = "error";
 
     [HttpGet("{slug}/{page?}")]
-    public async Task<IActionResult> Index(string slug, string page = "", bool isPreview = false,
-        [FromQuery] List<string>? tags = null)
+    public async Task<IActionResult> Index(string slug, string page = "", [FromQuery] List<string>? tags = null, CancellationToken cancellationToken = default)
```

Comment:
Is the removal of `isPreview` going to impact the content designers at all?

Reply:
I'll have to check with them to better understand their workflow for C&S stuff, but it should just bring it in line with existing PT content. The preview api will get all draft content from the PT Contentful env so content designers should just be able to preview content in staging